### PR TITLE
fix: `DenominatedOracle` divide by 0

### PR DESCRIPTION
**Pull request by xBA5ED**
Closes HAI-308

### CODE REVIEW

File: src/interfaces/oracles/IDenominatedOracle.sol [hunk 1]
```diff
@@ -8,6 +8,8 @@ interface IDenominatedOracle is IBaseOracle {
 
   /// @notice Throws if either the provided price source or denominated price source are null
   error DenominatedOracle_NullPriceSource();
+  /// @notice Throws if the feed price is invalid.
+  error DenominatedOracle_InvalidPriceFeed();
```
Comment (0xJabberwock) [inline src/interfaces/oracles/IDenominatedOracle.sol]: `IBaseOracle` already brings in such error. 😉

**Review by 0xJabberwock — CHANGES_REQUESTED** [review]
Small improvement proposal.

File: src/interfaces/oracles/IDenominatedOracle.sol (hunk 1 above)
Comment (xBA5ED) [reply src/interfaces/oracles/IDenominatedOracle.sol]: I thought I checked but I must have missed it, I removed the new one

**Review by 0xJabberwock — APPROVED** [review]

**Review by wei3erHase — APPROVED** [review]
👍 
(would have loved a test, but sounds good fix)

**Pull request merged**